src/wallet/interfaces.cpp

```diff
@@ -560,8 +560,12 @@ class WalletLoaderImpl : public WalletLoader
         options.create_flags = wallet_creation_flags;
         options.create_passphrase = passphrase;
         bilingual_str error;
-        util::Result<std::unique_ptr<Wallet>> wallet{MakeWallet(m_context, CreateWallet(m_context, name, /*load_on_start=*/true, options, status, error, warnings))};
-        return wallet ? std::move(wallet) : util::Error{error};
+        std::unique_ptr<Wallet> wallet{MakeWallet(m_context, CreateWallet(m_context, name, /*load_on_start=*/true, options, status, error, warnings))};
+        if (wallet) {
+            return wallet;
```

Comment:
... CI doesn't like this either.

```
wallet/interfaces.cpp:565:20: error: call to implicitly-deleted copy constructor of 'std::unique_ptr<interfaces::Wallet>'
            return wallet;
                   ^~~~~~
/tmp/cirrus-ci-build/ci/scratch/msan/build/include/c++/v1/memory:1553:3: note: copy constructor is implicitly deleted because 'unique_ptr<interfaces::Wallet>' has a user-declared move constructor
  unique_ptr(unique_ptr&& __u) _NOEXCEPT
  ^
./util/result.h:44:14: note: passing argument to parameter 'obj' here
    Result(T obj) : m_variant{std::in_place_index_t<1>{}, std::move(obj)} {}
             ^
```

I think this will work:

```suggestion
            return {std::move(wallet)};
```

Reply:
🤨